src/main/kotlin/org/malachite/estella/people/domain/JobSeekerPayloads.kt

```diff
@@ -2,10 +2,7 @@ package org.malachite.estella.people.domain
 
 import org.malachite.estella.commons.models.people.JobSeeker
 
-data class JobSeekerDTO(val user: UserDTO) {
-    companion object {
-        fun fromJobSeeker(jobSeeker: JobSeeker) = JobSeekerDTO(
-                UserDTO.fromUser(jobSeeker.user)
-        )
-    }
-}
\ No newline at end of file
+data class JobSeekerDTO(val user: UserDTO)
```

Comment:
Pytanie może trochę z dupki, ale czy `JobSeekerDTO` nie powinien mieć przypadkiem jeszcze innych rzeczy z job seekera? Takich jak id (przyda się na froncie?) i plików?

Reply:
Ok to dodam